content/en/docs/ops/integrations/spire/index.md

```diff
@@ -0,0 +1,393 @@
+    0 s:C = US, O = SPIRE, CN = httpbin-5f4d47c948-zm6st
+    i:C = US, O = SPIFFE
+    -----BEGIN CERTIFICATE-----
+    MIIC8DCCAdigAwIBAgIRANo+nhRtC6kTnbLUz1MelwcwDQYJKoZIhvcNAQELBQAw
+    HjELMAkGA1UEBhMCVVMxDzANBgNVBAoTBlNQSUZGRTAeFw0yMjAzMTAwMjMyMTVa
+    Fw0yMjAzMTAwMzMyMjVaMEAxCzAJBgNVBAYTAlVTMQ4wDAYDVQQKEwVTUElSRTEh
+    MB8GA1UEAxMYaHR0cGJpbi01ZjRkNDdjOTQ4LXptNnN0MFkwEwYHKoZIzj0CAQYI
+    KoZIzj0DAQcDQgAEMveAwlgnO7YjRZ4vKsAkCYZu3W0REs1nKu/XPEWEBzfht3hl
+    grziL3mLnECWeDnTdakjTnowESCsg3cidOpWEaOB0TCBzjAOBgNVHQ8BAf8EBAMC
+    A6gwHQYDVR0lBBYwFAYIKwYBBQUHAwEGCCsGAQUFBwMCMAwGA1UdEwEB/wQCMAAw
+    HQYDVR0OBBYEFHnxGd7U2ADDw4GouBb0wPLorrHeMB8GA1UdIwQYMBaAFLnNFhHO
+    0Xsdte9uf3VLILKA7TGtME8GA1UdEQRIMEaCGGh0dHBiaW4tNWY0ZDQ3Yzk0OC16
+    bTZzdIYqc3BpZmZlOi8vZXhhbXBsZS5vcmcvbnMvZGVmYXVsdC9zYS9odHRwYmlu
+    MA0GCSqGSIb3DQEBCwUAA4IBAQA6UZagZrw/htbpD7HbcmD17sQ2xv1+nMrRFZGZ
+    M9bO924uyz9MiEf/iR3jKD+wXrJdmyjJ+5Mas4QGnggaGUfDmE2wrVlFKIqY8R1E
+    hZonQm8YeWN4CtDijyAxHihgKTV2V2inrHaaqp1r2/Upt3/4XrZf1f1eqpl03hT9
+    +OPu0SV8MUPTPAXqaIPfjmV9fTTMYQg4g4smsijVKAl6lVs+PsZiqbcgEeUnT2JL
+    lel5nNXdXqKhygfqKesI30BERGPvZ5vdMtUrGxe0L5CPJrkcrtn/vRTuMsXWSr4m
+    VAw3TEEXVdnd3VuAIsKO+6J7biMZH6jhm/sPUqTrdjH/L7S7
+    -----END CERTIFICATE-----
+    ---
+    Server certificate
+    subject=C = US, O = SPIRE, CN = httpbin-5f4d47c948-zm6st
+    
+    issuer=C = US, O = SPIFFE
+   ...
+   {{< /text >}}
+
+## SPIFFE Federation
+
+SPIRE Servers are able to authenticate SPIFFE identities originated from different trust domains, this is known as SPIFFE federation.
+SPIRE Agent can be configured to push federated bundles to Envoy through the Envoy SDS API, allowing Envoy to use [validation context](https://spiffe.io/docs/latest/microservices/envoy/#validation-context)
+to verify peer certificates and trust a workload from another trust domain.   
+To enable Istio to federate SPIFFE identities through SPIRE integration:
+
+* Consult [SPIRE Agent SDS configuration](https://github.com/spiffe/spire/blob/main/doc/spire_agent.md#sds-configuration) and set the following 
+SDS configuration values for your SPIRE Agent configuration file.
+
+| Configuration              | Description                                                                                      | Resource Name |
+|----------------------------|--------------------------------------------------------------------------------------------------|---------------|
+| `default_svid_name`        | The TLS Certificate resource name to use for the default X509-SVID with Envoy SDS                | default       |
+| `default_bundle_name`      | The Validation Context resource name to use for the default X.509 bundle with Envoy SDS          | null          |
+| `default_all_bundles_name` | The Validation Context resource name to use for all bundles (including federated) with Envoy SDS | ROOTCA        |
+
+This will allow Envoy to get federated bundles directly from SPIRE.
+
+### Create Federated Registration Entries

```

Comment:
The [Istio Style Guide](https://istio.io/latest/docs/releases/contribute/style-guide/#use-sentence-case-for-headings) recommends sentence case for headings.
```suggestion
### Create federated registration entries

```